tools/e2e/setup.mjs

```diff
@@ -88,26 +95,21 @@ export const startRegistry = async () => {
  * }
  */
 export const setup = async () => {
-  const { storage, url } = await startRegistry()
+  const { url } = await startRegistry()
   const workspace = await mkdtemp(`${tmpdir()}${sep}e2e-test-`)
 
-  const npmrc = join(homedir(), '.npmrc')
+  const npmrc = fileURLToPath(new URL('../../.npmrc', import.meta.url))
```

Comment:
the `.npmrc` file has to be always in the homedir and depending on your checkout location this will not work

Reply:
This is the location of the `.npmrc` file that is committed to the repository. During publishing this adds the auth token to it.